src/hammer-vlsi/defaults.yml

```diff
@@ -595,6 +615,20 @@ sram_generator.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+# PCB deliverable settings
+pcb.inputs:
+
+  # Desired pad opening diameter
+  # Typically this should be defined in the technology plugin, assuming everyone will use the same pad and board strategy
+  # type: Decimal
+  bump_pad_opening_diameter: null
```

Comment:
We should note whether these are in drawn or real units. I think convention is board stuff -> real units, chip/PNR stuff -> drawn but you would know better than me.

Reply:
good point. these should be post-shrink (eventually we'll combine them in #378)

Reply:
I clarified this in the comment